core/src/main/java/org/apache/oozie/action/hadoop/MapReduceActionExecutor.java

```diff
@@ -474,6 +474,13 @@ protected String findYarnApplicationId(final Context context, final WorkflowActi
         }
     }
 
+    @Override
+    protected void addActionSpecificEnvVars(Map<String, String> env) {
```

Comment:
It will be compatible with previous Hadoop version? Like hadoop 2.6?

Reply:
Yes, that's right